Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.11.0-alpine
+LABEL authors="serhiik"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /cinema_service
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
Copying the entire context with `COPY . .` can lead to larger image sizes if unnecessary files are included. Ensure your `.dockerignore` file is properly configured to exclude files that are not needed in the final image.